src/main/java/ua/danit/final_project/entities/Position.java

```diff
@@ -35,6 +35,8 @@
   @Column(name = "p_title", nullable = false, unique = true)
   private String title;
 
+  private Boolean pinnedToComment;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "pinnedToComment" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=104&issues=AWW5JKvo_eEZPtUmV0Xm&open=AWW5JKvo_eEZPtUmV0Xm)

[ISSUE_KEY]: AWW5JKvo_eEZPtUmV0Xm